server-mysql/README.md

```diff
@@ -14,7 +14,7 @@ First start a MySQL instance using the MySQL docker image:
 
 Start a Keycloak instance and connect to the MySQL instance:
 
-    docker run --name keycloak --link mysql:mysql jboss/keycloak-mysql
+    docker run -it -p 8080:8080 --name keycloak --link mysql:mysql jboss/keycloak-mysql
```

Comment:
I would better not commit this, but maybe @mstruk or @abstractj can say what they prefer?